build.gradle

```diff
@@ -241,10 +241,10 @@ task extractSchema(type: Sync) {
  *********************************************************************/
 
 //These are the tasks available in the Schema dependency that we would want to use in this project
-def taskMap = ["migrateReporting": "Migrates the schemas that are required for running locally",
-               "cleanReporting": "Cleans the schemas that are required for running locally",
-               "migrateReporting_test": "Migrates the schemas that integration tests depend on",
-               "cleanReporting_test": "Cleans the schemas that integration tests depend on"]
+def taskMap = ["migrateWarehouse": "Migrates the schemas that are required for running locally",
+               "cleanWarehouse": "Cleans the schemas that are required for running locally",
+               "migrateWarehouse_test": "Migrates the schemas that integration tests depend on",
```

Comment:
I think in reporting isn't it `migrateReportingTest` ?  We should be consistent here.